plugins/quota/validate.go

```diff
@@ -3,38 +3,65 @@ package quota
 import (
 	"fmt"
 
+	daemon "github.com/gluster/glusterd2/glusterd2/daemon"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/errors"
 )
 
 const (
-	quotaEnabledKey = "features.quota"
+	quotaEnabledKey = "quota.enable"
 )
 
 func validateOptions(v *volume.Volinfo, key string, value string) error {
 
-	if isQuotaEnabled(v) == false {
-		err := fmt.Errorf("Quota not enabled to set this value: '%s'", key)
-		return err
+	// Check if volume is started
+	if v.State != volume.VolStarted {
+		return errors.ErrVolNotStarted
 	}
 
 	switch key {
-	case "deem-statfs":
+	case "enable":
+		switch value {
+		case "on":
+			ret, err := isQuotadRunning()
+			if err == nil && ret {
```

Comment:
I think we can check only error condition here, no need to check ```ret```

Reply:
done